integrations/clickup/clickup/handler.py

```diff
@@ -0,0 +1,65 @@
+from loguru import logger
+from clickup.client import ClickUpClient
+from port_ocean.context.ocean import ocean
+
+class ClickUpPortHandler:
+    def __init__(self):
+        self.client = self.get_clickup_client()
+
+    def get_clickup_client(self):
+        try:
+            host = ocean.integration_config["clickup_host"]
+            api_key = ocean.integration_config["clickup_api_key"]
+            if not host or not api_key:
+                raise ValueError("ClickUp host or API key is not configured properly.")
+            return ClickUpClient(host, api_key)
+        except KeyError as e:
+            logger.error(f"Configuration key missing: {e}")
+            raise
+
+    async def fetch_teams(self):
+        try:
+            async for teams in self.client.get_teams():
+                yield teams
+        except Exception as e:
+            logger.error(f"Failed to fetch teams: {e}")
+            raise
+
+    async def fetch_spaces(self, team_id: str):
+        try:
+            async for spaces in self.client.get_spaces(team_id):
+                yield spaces
+        except Exception as e:
+            logger.error(f"Failed to fetch spaces for team {team_id}: {e}")
+            raise
+
```

Comment:
this handler file is making the code complex. Just use the functions in client.py directly in main.py. You don't need another layer